assets/js/header.js

```diff
@@ -18,6 +18,16 @@ if (mainInner !== null) {
 headerInner.style.setProperty('opacity', 1);
 
 
+// Measure header height for the scrolling fix
+{
```

Comment:
The block here is necessary to avoid declaring constants at the top level of the script - this will conflict with another declaration of `const header` elsewhere. It might generally be a good idea to wrap each script file in a block to give it its own scope - this will make sure there are no conflicts between declarations in different script files.